File: web3rpc/rpc-specs/code-samples/python/eth/filter/uninstallFilter.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

filterId = "0xb"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.uninstall_filter(filterId)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/filter/uninstallFilter.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const filterId = '0xb'

    sdk.eth.uninstallFilter(filterId, {}, (err, data, response) => {
        console.log(data);
    });
}
)()